lib/cdo/delete_accounts_helper.rb

```diff
@@ -289,16 +289,6 @@ def remove_email_preferences(email)
     @log.puts "Removed #{record_count} EmailPreference" if record_count > 0
   end
 
-  # Removes signature and school_id from applications for this user
-  # @param [User] user
-  def anonymize_circuit_playground_discount_application(user)
-    @log.puts "Anonymizing CircuitPlaygroundDiscountApplication"
-    if user.circuit_playground_discount_application
-      user.circuit_playground_discount_application.anonymize
-      @log.puts "Anonymized 1 CircuitPlaygroundDiscountApplication"
-    end
```

Comment:
This can be deleted since the immediate followup PR for this will drop the `CircuitPlaygroundDiscountApplication` table